agent/cmd/agent/main.go

```diff
@@ -0,0 +1,191 @@
+package agent
+
+import (
+	"context"
+	"crypto/tls"
+	"flag"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/buoyantio/linkerd-buoyant/agent/pkg/api"
+	"github.com/buoyantio/linkerd-buoyant/agent/pkg/bcloudapi"
+	"github.com/buoyantio/linkerd-buoyant/agent/pkg/handler"
+	"github.com/buoyantio/linkerd-buoyant/agent/pkg/k8s"
+	pb "github.com/buoyantio/linkerd-buoyant/gen/bcloud"
+	l5dApi "github.com/linkerd/linkerd2/controller/gen/client/clientset/versioned"
+	"github.com/linkerd/linkerd2/pkg/admin"
+	l5dk8s "github.com/linkerd/linkerd2/pkg/k8s"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog/v2"
+
+	// Load all the auth plugins for the cloud providers.
+	// This enables connecting to a k8s cluster from outside the cluster, during
+	// development.
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
```

Comment:
move this into `agent/main.go`

https://github.com/golang/go/wiki/CodeReviewComments#import-blank